fenixedu-ist-giaf-invoices/src/main/java/pt/ist/fenixedu/giaf/invoices/ErrorLogConsumer.java

```diff
@@ -0,0 +1,22 @@
+package pt.ist.fenixedu.giaf.invoices;
+
+@FunctionalInterface
+public interface ErrorLogConsumer {
+
+    void accept(final String oid,
```

Comment:
You could still use the same basic principle of passing along all that information to a consumer. The difference would be to move the arguments to an object with some sort of fluent API to make things more readable, and change your custom consumer to a `Consumer<ThatType>`.

But given the low impact of this code in the rest of the system, just leave it as it is.